.eslintignore

```diff
@@ -1,2 +1,5 @@
-coverage/*
+__tests__/
+*.jsx
+packages/*/*.js
```

Comment:
Looks like this will disable ESLint for most of the files ;-/

Reply:
Sorry my bad. Fixed in b0c14e7231d3fa7471fbaae270b1e9db5102a46c

Reply:
The pattern is still here ;-/

Reply:
Ok sorry I wasn't sure which line you were referring to. Actually it only disable linting on js files at the root of packages, which are actually files built by babel. Same logic as with `.gitignore`.